models/bernoulli_connection.h

```diff
@@ -58,8 +58,8 @@ Parameters
 
 FirstVersion: June 2017
 
-Transmits
-+++++++++
+Sends
```

Comment:
This needs to stay `Transmits`. Also in other files ending in `_connection.h`.

The logic is that neurons have the (single) event type they will send during the simulation under `Sends` and all event types they are able to handle under `Receives`. As synapses/connections only relay the events from the sender to the receiver, the have everything they can relay under `Transmits`.